modules/c++/sys/source/OSUnix.cpp

```diff
@@ -189,6 +189,7 @@ std::string sys::OSUnix::getTempName(const std::string& path,
     else
     {
         name = tempname;
+        fclose(fopen(name));
```

Comment:
Consistent with Window and Linux, yes. This path is only taken if we don't have Win32 or GCC. The other two cases create a file.